video_xblock/tests/unit/test_backends.py

```diff
@@ -140,6 +143,10 @@ def test_get_player_html(self):
             'start_time', 'end_time', 'handout',
             'download_transcript_allowed', 'download_video_allowed',
         ],
+        [  # Tencent
+            'start_time', 'end_time', 'handout',
+            'download_transcript_allowed', 'download_video_allowed', 'download_video_url'
+        ],
```

Comment:
Let's please don't display advanced fields as we haven't implemented such functionality, thanks.